posthog/test/test_team_model.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+
+from ..models.team import Team
+
+
+class TestTeam(TestCase):
+    def test_team_str(self):
+        # 1. Use the team.name by default
+        team: Team = Team.objects.create(name="The Mighty Library")
+        self.assertEqual(str(team), "The Mighty Library")
+
+        # 2. If the team has no name:
+
+        # 2.1 Team has one app_url
+        team = Team.objects.create(app_urls=["app.posthog.com"])
+        self.assertEqual(str(team), "app.posthog.com")
+
+        # 2.2 Team has multiple app_urls
```

Comment:
I like the summary comments, but adding numbers/sectioning implies that these checks must be done in steps/sections.

It is also more of a pain to maintain (eg if you want to insert a new section, you have to go back and rename all the previous numbers)